zkevm-circuits/src/copy_circuit.rs

```diff
@@ -787,16 +796,12 @@ impl<F: Field> CopyCircuitConfig<F> {
                     .iter()
                     .zip_eq(table_row)
             {
-                // Leave sr_addr_end and bytes_left unassigned when !is_read
-                if !is_read && (label == "src_addr_end" || label == "bytes_left") {
```

Comment:
Removing an unecessary exceptional case, also removing logic from the assign function.